modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -97,6 +101,7 @@ interfaces:
   ipv4:
     enabled: true
     dhcp: true
+    auto-route-metric: 48 <6>    
   ipv6:
     enabled: false
     dhcp: false
```

Comment:
Okay, I took a vow last year to reject any single-stack example.  :smiley:   With the IPv6 mandates coming we can't keep omitting IPv6.

All examples should include IPv6, or we just have to fix it later.  

We can just copy the IPv4 section. 

```
    ipv6:
      enabled: true
      dhcp: true
      auto-route-metric: 48
```

This case is important, because the `auto-route-metric` needs to be set for both, so it's valuable to include it in the example.

Reply:
:heavy_check_mark: